src/content/developers/docs/gas/index.md

```diff
@@ -21,88 +20,104 @@ Since each Ethereum transaction requires computational resources to execute, eac
 
-Total fee will be: `Gas units (limit) * Gas price per unit`
+Total fee would have been: `Gas units (limit) * Gas price per unit`
 i.e `21,000 * 200 = 4,200,000 gwei` or 0.0042 ETH
 
-Now, when Alice sends the money, 1.0042 ETH will be deducted from Alice's account.
-Bob will be credited 1.0000 ETH.
-Miner gets 0.0042 ETH.
+When Alice sent the money, 1.0042 ETH would be deducted from Alice's account.
+Bob would be credited 1.0000 ETH.
+Miner would receive 0.0042 ETH.
 
 This video offers a concise overview of gas and why it exists:
 
 <iframe width="100%" height="315" src="https://www.youtube.com/embed/AJvzNICwcwc" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## Why do gas fees exist? {#why-do-gas-fees-exist}
+## After to the London upgrade {#post-london}
 
-In short, gas fees help keep the Ethereum network secure. By requiring a fee for every computation executed on the network, we prevent actors from spamming the network. In order to prevent accidental or hostile infinite loops or other computational wastage in code, each transaction is required to set a limit to how many computational steps of code execution it can use. The fundamental unit of computation is "gas".
+Starting with [the London update](/history/#london), every block has a base fee, the minimum per gas price for inclusion in this block. Optionally, users can set a priority fee (tip).
 
-Although a transaction includes a limit, any gas not used in a transaction is returned to the user.
+Calculating the total fee works as follows: `Gas units (limit) * (Base fee + Tip)`
 
-![Diagram showing how unused gas is refunded](../transactions/gas-tx.png)
-_Diagram adapted from [Ethereum EVM illustrated](https://takenobu-hs.github.io/downloads/ethereum_evm_illustrated.pdf)_
+Let’s say Jordan has to pay Taylor 1 ETH. In the transaction the gas limit is 21,000 units and the base fee is 100 gwei. Jordan includes a tip of 10 gwei.
+Using the formula above we can calculate this as `21,000 * (100 + 10) = 2,310,000 gwei` or 0.0023 ETH.
 
-## What is gas limit? {#what-is-gas-limit}
+## Base Fees {#base-fees}
 
-Gas limit refers to the maximum amount of gas you are willing to consume on a transaction. More complicated transactions, involving [smart contracts](/developers/docs/smart-contracts/), require more computational work so they require a higher gas limit than a simple payment. A standard ETH transfer requires a gas limit of 21,000 units of gas.
+Every block has a base fee which acts as a reserve price. To be eligible for inclusion in a block the offered price per gas must at least equal the base fee. The base fee is calculated independently of the current block and is instead determined by the blocks before it - making transaction fees more predictable for users. When the block is mined this base fee is "burned", removing it from circulation.
```

Comment:
We should call out that the base fee is calculated by the protocol itself & is a value that's supplied by clients.

Reply:
Also worth mentioning that it fluctuates based on network demand?

Reply:
> We should call out that the base fee is calculated by the protocol itself & is a value that's supplied by clients.

Agreed

>Also worth mentioning that it fluctuates based on network demand?

It's stated explicitly in 'block size' paragraph but may make some sense to include here depending on the order we go with. I'll revisit after looking at the ordering of paragraphs as referenced here [in this comment](https://github.com/ethereum/ethereum-org-website/pull/3509#discussion_r682166844).